editor/editor.tsx

```diff
@@ -312,20 +308,9 @@ function Editor(props: RouteComponentProps) {
     useEffect(() => {
         previewData.current = [];
         setSelectedPreviewData(0);
-        if (urlExampleId && !validateExampleId(urlExampleId)) {
-            // invalida url example id
-            setCode(emptySpec(`Example id "${urlExampleId}" does not exist.`));
-            setJsCode(emptySpec(`Example id "${urlExampleId}" does not exist.`));
-        } else if (urlSpec) {
-            setCode(urlSpec);
-            setJsCode(json2js(urlSpec));
-        } else if (urlGist) {
-            setCode(emptySpec());
-        } else {
-            const jsonCode = stringifySpec(demo.spec as gosling.GoslingSpec);
-            setCode(jsonCode);
-            setJsCode(demo.specJs ?? json2js(jsonCode));
-        }
+        const jsonCode = stringifySpec(demo.spec as gosling.GoslingSpec);
+        setCode(jsonCode);
+        setJsCode(demo.specJs ?? json2js(jsonCode));
```

Comment:
I assume this change fixes a bug?

Reply:
yes, it fixes a bug. Specifically, if you open the current gosling editor with the urlparams (e.g., ?example=xxxx), the editor will not response when you click demo in the example gallery.